src/shared/utils/flattenChildren.js

```diff
@@ -54,7 +54,10 @@ function flattenSingleChildIntoContext(
  * children will not be included in the resulting object.
  * @return {!object} flattened children keyed by name.
  */
-function flattenChildren(children: ReactElement<any>, selfDebugID: number): ?{ [name: string]: ReactElement<any> } {
+function flattenChildren(
+  children: ReactElement<any>,
+  selfDebugID?: number,
```

Comment:
What's the difference between `selfDebugID: ?number` and `selfDebugID?: number`?